packages/core/src/browser/xhrProxy.spec.ts

```diff
@@ -209,4 +237,50 @@ describe('xhr proxy', () => {
       },
     })
   })
+
+  it('should track multiple requests with the same xhr instance', (done) => {
+    let listeners: { [k: string]: Array<() => void> }
+    withXhr({
+      completionMode: 'manual',
+      setup(xhr, complete) {
+        const secondOnload = () => {
+          xhr.removeEventListener('load', secondOnload)
+          complete(xhr)
```

Comment:
```suggestion
      setup(xhr) {
        const secondOnload = () => {
          xhr.removeEventListener('load', secondOnload)
```
with this modification, the test still pass so I don't get what you are trying to achieve with manual mode.
could we discuss it?

Reply:
Yes, no longer needed